CHANGELOG.rst

```diff
@@ -2,22 +2,28 @@
 Changelog
 =========
 
-TBR
-------------------
+0.2.0 (TBD)
+-----------
+
+* Backward Incompatible Change:
+
+    * ``ResponseData`` is replaced with ``HttpResponse``.
+
+      ``HttpResponse`` exposes methods useful for web scraping
+      (such as xpath and css selectors, json loading),
+      and handles web page encoding detection. There are also new
+      types like ``HttpResponseBody`` and ``HttpResponseHeaders``.
 
 * Added support for performing additional requests using
   ``web_poet.HttpClient``.
+* Introduced ``web_poet.BrowserHtml`` dependency
```

Comment:
I just realized that we need to update https://raw.githubusercontent.com/scrapinghub/web-poet/master/docs/api_reference.rst to properly reflect it in the docs.